docs/program/extend/modular-resources/examples/add-tflite-module.md

```diff
@@ -0,0 +1,325 @@
+
+## Generate your robot configuration
+
+When you built the C++ SDK, the build process also built the `example_audio_classification_client` binary, which includes a `--generate` function that determines and creates the necessary robot configuration to support this example.
+
+To generate your robot's configuration using `example_audio_classification_client`:
+
+1. First, determine the full path to the `yamnet/classification` model you just downloaded.
+   If you followed the instructions above, this path is: <file>~/example_workspace/lite-model_yamnet_classification_tflite_1.tflite</file>.
+
+1. Next, determine the full path to the `example_mlmodelservice_tflite` modular resource example provided with the Viam C++ SDK.
+   If you followed the instructions above, this path is: <file>~/example_workspace/opt/bin/example_mlmodelservice_tflite</file>.
+
+1. Run the `example_audio_classification_client` binary, providing both paths to the `--generate` function in the following fashion:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   cd ~/example_workspace/opt/bin
+   ./example_audio_classification_client --generate --model-path ~/example_workspace/lite-model_yamnet_classification_tflite_1.tflite --tflite-module-path ~/example_workspace/opt/bin/example_mlmodelservice_tflite > ~/example_workspace/viam-example-mlmodel-config.json
+   ```
+
+1. Verify that the resulting configuration file was created successfully:
+
+   ```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+   cat ~/example_workspace/viam-example-mlmodel-config.json
+   ```
+
+1. Copy the contents of this file.
+   Then return to your robot's page on [the Viam app](https://app.viam.com), select the **Config** tab, select **Raw JSON**, and paste the configuration into the text area.
+
+1. Click the **Save config** button at the bottom of the page.
+   Now, when you switch back to **Builder** mode, you can see the new configuration settings under the **Services** and **Modules** subtabs.
+
+This generated configuration features the minimum required configuration to support this tutorial: `services` parameters for the [ML model](/services/ml/) service and `modules` parameters for the `example_mlmodelservice_tflite` module.
+
+## Run the inference client
+
+With everything configured and running, you can now run the inference client that connects to `viam-server` and uses the `example_mlmodelservice_tflite` module.
+
+1. First, determine your robot address and location secret. To do so, navigate to [the Viam app](https://app.viam.com), select the **Code sample** tab, and toggle **Include secret**.
+   The location secret resembles `abcd1234abcd1234abcd1234abcd1234abcd1234abcd1234` and the robot address resembles `my-robot-main.abcdefg123.viam.cloud`.
+
+   {{%  snippet "secret-share.md" %}}
+
+1. Then, run the following to start the inference client, providing the necessary access credentials and the path to the labels file extracted earlier:
```

Comment:
Aren't we meant to discourage folks from putting their secret and address into their command line history?